brands/BSA/tokens/index.js

```diff
@@ -440,6 +440,31 @@ export default {
 		},
 	},
 
+	list: {
+		bullet: {
+			appearance: {
+				primary: {
+					color: COLORS.primary,
+				},
+				hero: {
+					color: COLORS.hero,
+				},
+				neutral: {
+					color: COLORS.neutral,
+				},
+			},
+		},
+		link: {
+			color: COLORS.primary,
+		},
+		tick: {
+			color: COLORS.primary,
+		},
+		icon: {
+			color: COLORS.primary,
+		},
+	},
+
```

Comment:
we could wrap these in `type:` to be consistent; the prop we're keying against is the type prop. Thought?

```
list: {
	type: {
		bullet: {
			appearance: {
				primary: {
					color: COLORS.primary,
				},
				hero: {
					color: COLORS.hero,
				},
				neutral: {
					color: COLORS.neutral,
				},
			},
		},
		link: {
			color: COLORS.primary,
		},
		tick: {
			color: COLORS.primary,
		},
		icon: {
			color: COLORS.primary,
		},			
	}
},
```

Reply:
I'll change it to this for consistency but I  do think we need to discuss what our standard structure for tokens are and document it. 

i.e. whether we should be doing it
```
list[type]
```
vs
```
list.type[type]
```

I personally think that `list[type]` is nicer since we aren't repeating ourselves and reduces nesting but happy to discuss.

Reply:
Other examples such as `button[appearance]` vs `button.appearance[appearance]` to be considered too

There is an added benefit to using the more verbose approach which is it often helps with spreading the CSS properties (our kind of CSS setting shorthand). But yeah definitely need to discuss decide on a token structure.

The appearance key nested inside the type key definitely feels a bit clunky